doc/doc-site/docs/overview/orders.md

```diff
@@ -261,7 +261,7 @@ Here, `APT` is considered the "base" asset and `USDC` is considered the "quote a
 | Quote asset | Order price | `USDC` | 6             | \$1.00     |
 
 We'd like to find a market configuration that makes sense given the above economic state.
-Configuring a market requires three values beyond the base and quote type:
+Configuring a market requires three _integer_ values beyond the base and quote type:
```

Comment:
As requested in the original comment, https://github.com/econia-labs/econia/pull/440#discussion_r1304854242, how about something in here that indicates these values are expected by Move?

"Configuring a market requires three _integer_ values beyond the base and quote type, since the Econia Move package uses only integer arithmetic."

Reply:
done [e1f079a](https://github.com/econia-labs/econia/pull/440/commits/e1f079af7b2a0bcad8e0af94742a6d2fdf1eb589)